wiki/Tournaments/Mirai/2020/en.md

```diff
@@ -0,0 +1,570 @@
+  - [Streetlight Manifesto - Punk Rock Girl \[Insane\]](https://osu.ppy.sh/beatmapsets/577394#osu/1269449)
+  - [BlackY vs. Yooh - HAVOX \[Ayyri's EXHAUST\]](https://osu.ppy.sh/beatmapsets/823616#osu/1876631)
+- Hidden
+  - [TAMAONSEN - Witches' Night (ShinRa-Bansho Ver.) \[Dafiely & Xenon's Insane\]](https://osu.ppy.sh/beatmapsets/1030100#osu/2165811)
+  - [Yooh - snow storm -euphoria- \[EXHAUST\]](https://osu.ppy.sh/beatmapsets/158075#osu/420012)
+- HardRock
+  - [Amane - Boozehound \[Nuts\]](https://osu.ppy.sh/beatmapsets/5875#osu/27810)
+  - [mafumafu - Kakushigoto \[NiNo's Light Insane.\]](https://osu.ppy.sh/beatmapsets/770023#osu/1618984)
+- DoubleTime
+  - [SteLouse - Joyride \[Hard\]](https://osu.ppy.sh/beatmapsets/747507#osu/1575098)
+  - [yuikonnu - Yume no Mata Yume \[Hard\]](https://osu.ppy.sh/beatmapsets/812160#osu/1703566)
+  - [Dictate - Militant \[Hard\]](https://osu.ppy.sh/beatmapsets/785677#osu/1649483)
+- ForceMod
+  - [Ghost - From The Pinnacle To The Pit \[Insane\]](https://osu.ppy.sh/beatmapsets/425520#osu/919002)
+  - [XL Project - Necro Fantasia \[Collab\]](https://osu.ppy.sh/beatmapsets/23077#osu/79266)
+- Tiebreaker
+  - **[Halozy - Paranoid Lost \[Soda's Lunatic\]](https://osu.ppy.sh/beatmapsets/440925#osu/964622)**
+
+### Qualifiers
+
+- NoMod
+  - [FELT - Story \[Lunatic\]](https://osu.ppy.sh/beatmapsets/297409#osu/944709)
+  - [Street - Sakura Fubuki \[Another\]](https://osu.ppy.sh/beatmapsets/362989#osu/827019)
+  - [Sakuzyo - FAFNIR \[KKip's Insane\]](https://osu.ppy.sh/beatmapsets/777255#osu/1637690)
+  - [Streetlight Manifesto - The Blonde Lead The Blind \[Insane\]](https://osu.ppy.sh/beatmapsets/376545#osu/826347)
+- Hidden
+  - [Saratoga (CV: Uesaka Sumire) - Souzetsu Gekkou \[Insane\]](https://osu.ppy.sh/beatmapsets/765452#osu/2178006)
+  - [Veil of Maya - Mikasa \[Insane\]](https://osu.ppy.sh/beatmapsets/480298#osu/1219509)
+- HardRock
+  - [Celldweller - Good L*ck (Yo*'re F_cked) \[Hard\]](https://osu.ppy.sh/beatmapsets/1069112#osu/2239676)
+  - [Zektbach - meme \[Hard\]](https://osu.ppy.sh/beatmapsets/68617#osu/199410)
+- DoubleTime
+  - [senya - Terasareru kurai no Shiawase \[Collab Hard\]](https://osu.ppy.sh/beatmapsets/747507#osu/1575098)
+  - [Creo - Rivals \[deetz' Hard\]](https://osu.ppy.sh/beatmapsets/1012878#osu/2142929)
+
+---
+
+## Match results
+
+### Grand Finals
+
+|            Blue team |       |     | Red team  | Match link                                          |
+| -------------------: | :---: | :-: | :-------- | :-------------------------------------------------- |
+| **Norank MaMa <3 U** | **7** |  3  | Uprankers | [#1](https://osu.ppy.sh/community/matches/63280403) |
+
+### Losers Finals
+
+|     Blue team |     |     | Red team    | Match link       |
+| ------------: | :-: | :-: | :---------- | :--------------- |
+| **Uprankers** |  0  | -1  | Salted_Fish | _win by default_ |
```

Comment:
```suggestion
| **Uprankers** |  0  | -1  | Salted_Fish | *win by default* |
```
Use asterisks instead of underscores, apply for; win by defaults, nullified